src/test/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherTaxServiceImplTest.java

```diff
@@ -0,0 +1,291 @@
+package edu.cornell.kfs.fp.document.service.impl;
+
+import java.util.Collection;
+import java.util.List;
+
+import junit.framework.TestCase;
+
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.vnd.businessobject.VendorAddress;
+import org.kuali.kfs.vnd.businessobject.VendorContract;
+import org.kuali.kfs.vnd.businessobject.VendorDetail;
+import org.kuali.kfs.vnd.businessobject.VendorHeader;
+import org.kuali.kfs.vnd.businessobject.VendorRoutingComparable;
+import org.kuali.kfs.vnd.document.service.VendorService;
+import org.kuali.rice.core.api.parameter.ParameterEvaluator;
+import org.kuali.rice.core.api.parameter.ParameterEvaluatorService;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.bo.Note;
+import org.kuali.rice.krad.document.Document;
+
+public class CuDisbursementVoucherTaxServiceImplTest extends TestCase {
```

Comment:
I would suggest adding a few test cases where the DV payee is not a vendor (such as an employee payee).